example/joint/glove_donning.py

```
@@ -0,0 +1,43 @@
+"""穿脱手套 demo: 将所有关节平滑归零（手掌摊平），便于穿戴或脱下手套。
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)